docs/getting_started/basics/execute.md

```diff
@@ -34,22 +36,65 @@ The "Execute Next" action will compute the next dirty node present in the workfl
 </figure>
 
 
-## ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
+### ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
 
-The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed. 
+The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed.
```

Comment:
I would extend about how this button behaves if there's a workflow currently executing

Reply:
Added one line to explain that

Reply:
Have you pushed? I don't see it.

Reply:
now it should be there